charts/argo/templates/workflow-crd.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Since resource name is not parameterized helm won't allow installing two argo releases:

```
$ helm install --namespace kube-system -n new-release .
Error: release new-release failed: customresourcedefinitions.apiextensions.k8s.io "workflows.argoproj.io" already exists
```

As workaround we could use pre-install hook. Hook should manually install the resource using `kubectl apply -f workflow-resources-config.yaml`